.github/workflows/publish.yml

```diff
@@ -1,12 +1,19 @@
 name: Publish
 
+run-name: "Publish site. ENV:${{ github.event.inputs.site_mode }}"
+
 on:
   workflow_dispatch:
     inputs:
       site_mode:
         description: Site mode can be either "production", "testing" or "development".
         required: true
-        default: testing
+        default: 'testing'
+        type: choice
+        options:
+        - testing
+        - development
+        - production
```

Comment:
Please, rebase to `feature/11ty`, cause these changes are [already there](https://github.com/ONLYOFFICE/api.onlyoffice.com/commit/1f33f58f8fdfe0db8e1f2094eba11ff6ee7d5941).